dashboard/src/containers/RoutesContainer/RoutesContainer.tsx

```diff
@@ -4,8 +4,8 @@ import { withRouter } from "react-router";
 import { IStoreState } from "../../shared/types";
 import Routes from "./Routes";
 
-function mapStateToProps({ namespace }: IStoreState) {
-  return { namespace: namespace.current };
+function mapStateToProps({ auth, namespace }: IStoreState) {
+  return { namespace: namespace.current || auth.defaultNamespace };
```

Comment:
[Not important] I understand the `defaultNamespace` is associated to the user and it's fetched during the authentication process. However, I'd expect to be defined as a property of the `namespace` redux state key.

Reply:
Yeah - I did initially think of going that way as I thought it would be natural to get it from the `state.namespace` key, but it is the responsibility of the `state.auth` (and associated reducer). So putting it in `state.namespace` would require the namespace reducer to handle the `auth.setAuthenticated` action - which is easy to do, but seemed odd to reach in like that (I see we do it for more generic things, like when the LOCATION changes which the namespace cares about).

In the end, it seemed quite natural here to use either the current namespace or the default one from the authorization data, but happy to change it if others think otherwise too.

Reply:
No problem. As you mentioned, `auth` is the responsible of fetching that data, so it's fine to keep it in the `state.auth` object too :)